zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/Notebook.java

```diff
@@ -414,7 +423,8 @@ public Note loadNoteFromRepo(String id, AuthenticationInfo subject) {
         for (AngularObject object : objectList) {
           SnapshotAngularObject snapshot = angularObjectSnapshot.get(object.getName());
           if (snapshot == null || snapshot.getLastUpdate().before(lastUpdatedDate)) {
-            angularObjectSnapshot.put(object.getName(),
+            angularObjectSnapshot.put(
```

Comment:
Are these changes caused by IDE automatically ?

Reply:
Yes, it is.

Reply:
Could you remove these changes caused by IDE ? Otherwise it is difficult to track the commit history.

Reply:
I don't set any special things except setting google-java-format. Could you try to do it with the same setting?